DevTools/Common/Models/Camunda/Authorization/AuthorizationGetListModels.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections.Generic;
```

Comment:
Add one more example for single entity + make response as transformable (`response` => `entity`)

Reply:
The idea has some major issues to be implemented right now, postponed